bedrock/firefox/templates/firefox/features/pdf-complete-fr.html

```diff
@@ -0,0 +1,94 @@
+    <p>
+      {{ resp_img(
+        url="img/firefox/features/edit-pdf/fr-edit-pdf-ouvrir.png",
+        srcset={
+            "img/firefox/features/edit-pdf/fr-edit-pdf-ouvrir-high-res.png": "1.5x"
+        },
+        optional_attributes={
+          "alt": "",
+          "width": "600"
+        }
+      ) }}
+    </p>
+  </li>
+  <li>
+    <p><strong>Renseigner vos informations :</strong> Cochez  les cases appropriées à l’aide de l’outil Dessin symbolisé par un crayon.</p>
+  </li>
+  <li>
+    <p><strong>Pour ajouter du texte :</strong> cliquez sur l'icône Texte pour ajouter du texte tels que votre numéro fiscal, vos noms, adresse et autre renseignements fiscaux. </p>
+    <p>
+      {{ resp_img(
+        url="img/firefox/features/edit-pdf/fr-edit-pdf-texte.png",
+        srcset={
+            "img/firefox/features/edit-pdf/fr-edit-pdf-texte-high-res.png": "1.5x"
+        },
+        optional_attributes={
+          "alt": "",
+          "width": "600"
+        }
+      ) }}
+    </p>
+  </li>
+  <li>
+    <p>Vous pouvez déplacer n’importe quel de vos blocs de texte et ajuster la taille, comme bon vous semble. Vous pouvez facilement passer d’une page à l’autre de votre déclaration ou avoir une vue d’ensemble en sélectionnant dans les Outils. </p>
+    <p>{{ resp_img(
+        url="img/firefox/features/edit-pdf/fr-edit-pdf-modifier.png",
+        srcset={
+            "img/firefox/features/edit-pdf/fr-edit-pdf-modifier-high-res.png": "1.5x"
+        },
+        optional_attributes={
+          "alt": "",
+          "width": "600"
+        }
+      ) }}
+    </p>
+  </li>
+</ol>
+
+<p>Enregistrez le document une fois terminé et envoyez-le aux services fiscaux en suivant les instructions.</p>
+
+<p>Si vous souhaitez corriger une déclaration :  suivez les mêmes étapes pour ouvrir et <a href="{{ url('firefox.features.pdf-editor') }}">remplir votre PDF</a> avec Firefox. N'oubliez pas d'ajouter la mention "DÉCLARATION RECTIFICATIVE, ANNULE ET REMPLACE" en haut de votre formulaire. </p>
+
+<p>Faites les corrections nécessaires, enregistrez le nouveau fichier et envoyez-le selon les instructions.</p>
+
+<h2>Enregistrer sa déclaration au format PDF</h2>
+
+<p>L’éditeur PDF Firefox vous aide à remplir mais aussi à classer et archiver vos tâches administratives, en toute confidentialité.</p>
+
+<p>Vous pouvez organiser vos justificatifs comme les factures, les reçus de dons, et l’imprimé fiscal unique. Ces documents n'ont pas à être fournis lors de la déclaration mais doivent être conservés en cas de contrôle, pendant au moins 3 ans. Pourquoi ne pas se simplifier la tâche dès l’étape de la déclaration&nbsp;? </p>
+
+<p>Enregistrez tous vos PDF dans des dossiers bien nommés pour une gestion facile et sécurisée.</p>
+
+<hr>
+
+<h2>Vous n'avez pas encore Firefox ?</h2>
+<p><a href="{{ url('firefox.features.index') }}">En savoir plus</a> sur nos fonctionnalités, ou <a href="">Téléchargez Firefox</a> dès maintenant.</p>
```

Comment:
**Issue (blocking)** second anchor tag in this line is missing an HREF to the download firefox page

Reply:
✅